pkgs/development/compilers/graalvm/default.nix

```diff
@@ -0,0 +1,13 @@
+{ callPackage }:
+
+let
+  cePackages = callPackage ./community-edition { };
+in
+cePackages
+// rec {
+  buildGraalvm = callPackage ./buildGraalvm.nix;
+
+  graalvm-oracle_22 = callPackage ./graalvm-oracle { version = "22"; };
+  graalvm-oracle_17 = callPackage ./graalvm-oracle { version = "17"; };
```

Comment:
You are adding 2 separate versions but the commit only says `graalvm-oracle: init at 22`.

Reply:
I've split it up into two commits. I'll do some updates and introduce version 23 in a follow up PR